src/cart/crt_rpc.c

```diff
@@ -1427,7 +1427,6 @@ crt_common_hdr_init(struct crt_rpc_priv *rpc_priv, crt_opcode_t opc)
 	rpc_priv->crp_req_hdr.cch_xid = xid;
 
 	rpc_priv->crp_reply_hdr.cch_opc = opc;
-	rpc_priv->crp_reply_hdr.cch_rank = rank;
```

Comment:
with this removal, there seems to be no purpose to having the rank variable at all.
Is this meant to assign cch_dst_rank = rank? Or is it really not necessary (in which case the above variable and logic to assign it can be removed)?

Reply:
Removed rank and rc altogether from this call